### PR TITLE
RUN-16775 Upgrade cluster from 2.16 to 2.17 fail

### PR DESCRIPTION
**Pull request by kandelira**

### CODE REVIEW

File: docs/admin/runai-setup/self-hosted/k8s/upgrade.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ title: Upgrade self-hosted Kubernetes installation
 
 ## Upgrade Control Plane
 
+### Upgrade from Version 2.9, 2.13, 2.15 or 2.16
+
+Before upgrading the control plane, run:
+
+``` bash
+kubectl patch pv $POSTGRES_PV $THANOS_PV -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
```
[the remark this answers is not in the document]
Comment (jasonnovichRunAI) [reply docs/admin/runai-setup/self-hosted/k8s/upgrade.md:28]: @kandelira Can you answer Kfir?

[the remark this answers is not in the document]
Comment (kandelira) [reply docs/admin/runai-setup/self-hosted/k8s/upgrade.md:28]: added, thanks

**Comment by ozRunAI** [comment]
@roir @kandelira waiting for more clarification:

- Can it be solved without a new prerequisite?
- What happens if a customer did not run the command? What action needed?

cc @yarongol

**Comment by ozRunAI** [comment]
Thanks @lliranbabi for the details. Approved

**Pull request merged**